flexget/plugins/cli/templates.py

```diff
@@ -19,8 +19,11 @@ def list_file_templates(manager, options):
         else:
             plugin = '-'
         name = template_name.replace('.template', '').split('/')
```

Comment:
I think we can just go with this change, and skip all the rest. The result of a `str.split` is always a list, so no need to check that, and there will always be at least one part, because all the templates will have a name.
```suggestion
        name = template_name.replace('.template', '').split('/')[-1]
```